### PR TITLE
Add toggle

### PR DESCRIPTION
**Pull request by scoopex**

### CODE REVIEW

File: extension-files/tools/zabbix_discovery_devices [hunk 1]
```diff
@@ -157,10 +157,17 @@ parser.add_argument(
 
 parser.add_argument(
     '--only_rawdisk',
-    help='Discovery for real raw/hardware devices',
+    help='Discovery for real raw/hardware devices and partitions',
```
Comment (jeroenvermeulen) [inline extension-files/tools/zabbix_discovery_devices:160]: The `and partitions` in the Help does not match the be behaviour in https://github.com/zabbix-tooling/zabbix-agent-extensions/pull/51/commits/50eec03db54ccaaff5749513d9431d1ad56078c1#diff-f22d4c12383ab2ad75b41e7720997c17699c17e4237ce0ed82022005ecdd2fd2R274

**Review by jeroenvermeulen — CHANGES_REQUESTED** [review]

**Comment by jeroenvermeulen** [comment]
@scoopex Thank you for finding a way to be able to merge my PR!

**Review by jeroenvermeulen — DISMISSED** [review]
Thank you, discovery keys are more descriptive this way.

**Comment by jeroenvermeulen** [comment]
Accidentally approved the PR while https://github.com/zabbix-tooling/zabbix-agent-extensions/pull/51#pullrequestreview-2519649453 is still open, so I re-requested a review from myself.

File: extension-files/tools/zabbix_discovery_devices (hunk 1 above)
Comment (scoopex) [reply extension-files/tools/zabbix_discovery_devices]: Tnx, adapted the description.

**Review by jeroenvermeulen — APPROVED** [review]
Looks good

**Pull request merged**